components/Pagination/sub-components/ActionButton.jsx

```diff
@@ -16,18 +14,18 @@ const ActionButton = styled.a`
   user-select: none;
   white-space: nowrap;
   border-radius: 4px;
-  border: 2px solid transparent;
 
   ${({
     theme: {
       colors: {
         primary: { 100: primary100, 700: primary700 },
       },
-      spacing: { xsmall, xxsmall },
+      spacing: { xxxsmall, xsmall, xxsmall, medium },
```

Comment:
what do you think in we put the properties in order?

Reply:
Done